openshift.spec

```diff
@@ -132,9 +121,6 @@ do
     go install -ldflags "%{ldflags}" %{import_path}/cmd/openshift
 done
 
-# set the IMAGES
-sed -i 's|IMAGES=.*|IMAGES=%{docker_images}|' rel-eng/openshift-{master,node}.sysconfig
-
```

Comment:
I'm not certain what to do about this, if they have a config file as the services expect them to then passing this as a commandline argument is ignored, but then we have no way other than generating a config file to set the default registry.

If we care, perhaps this could be adjusted to patch pkg/cmd/util/variable/imagetemplate.go or override it at build time with an ldflag somehow?